delivery/router/router.js

```diff
@@ -1,24 +1,42 @@
 const express = require('express');
 const {processError} = require("./errors/error-handler");
-const {retrieveMeteorDataLastWeek, retrieveMeteorDataWithQueryParams} = require("../../use-cases/retrieve-meteors-data");
+const {
+  retrieveMeteorDataLastWeek, 
+  retrieveMeteorDataWithQueryParams,
+  getLinkOnRoverPicture
+} = require("../../use-cases/retrieve-meteors-data");
 require('dotenv').config();
 
 const router = express.Router();
+router.use(express.json());
 
 router.get('/meteors', (req, res) => {
   if (Object.keys(req.query).length !== 0) {
     retrieveMeteorDataWithQueryParams(res, req.query)
     .catch((err) => {
-      processError(err,res);
+      const processedError = processError(err,res);
```

Comment:
Please check our chat
I already described how to handle errors

Reply:
Thanks, redone .